dev/integration_tests/web/test/test.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html' as html;
+
+import 'package:web_integration/a.dart'
+  if (dart.library.io) 'package:web_integration/b.dart' as message1;
+import 'package:web_integration/c.dart'
+  if (dart.library.html) 'package:web_integration/d.dart' as message2;
+
+void main() {
+  String message;
+  if (message1.message == 'a' && message2.message == 'd')  {
+    message = '--- TEST SUCCEEDED ---';
+  } else {
+    message = '--- TEST FAILED ---';
+  }
+  print(message);
+  html.HttpRequest.request(
```

Comment:
Since you don't have a specialized `localhost` server that understands this protocol, this is not needed. I only needed this in the other test because for release and profile mode I didn't (couldn't) use `flutter run`.

Reply:
Ahh, perfect!